pages/curriculum.tsx

```diff
@@ -82,15 +82,28 @@ export const Curriculum: React.FC<GetAppProps> = ({ data }) => {
   })
   return (
     <Layout>
-      <div className="row">
+      <div className="row ">
         <AlertsDisplay alerts={alerts} page="curriculum" />
-        <div className="col-8">{lessonsToRender}</div>
-        <div className="col-4">
+        <div className="d-xl-flex col-xl-4 order-xl-2">
           <ProgressCard progressCount={progressPercentage} />
+        </div>
+        <div className="col-xl-8">{lessonsToRender.slice(0, 2)}</div>
+      </div>
+      <div className="row">
+        <div className="col-xl-8">{lessonsToRender.slice(2, 5)}</div>
+        <div className="d-none d-xl-flex col-xl-4">
           <AnnouncementCard announcements={announcements} />
+        </div>
+      </div>
+      <div className="row">
+        <div className="col-xl-8">{lessonsToRender.slice(5, 7)}</div>
+        <div className="d-none d-xl-block col-xl-4">
           <AdditionalResources />
         </div>
       </div>
+      <div className="row">
+        <div className="col-xl-8">{lessonsToRender.slice(7)}</div>
+      </div>
     </Layout>
```

Comment:
Sorry but this is bad. Instead of 2 divs you got a bunch of them now, and the layout also depends on the number of lessons, which I agree won't change on a short time, but when it changes then this will be another source of issues.

Reply:
the rest of the PR is great, I don't think its a good idea for annoucements to be aligned with each lesson... if announcement changes, you get weird white spaces.

Reply:
Announcement change is a problem yes. I used vertical flexbox so it should look fine as long as total announcement height won't become bigger than four lessons. As far as I know it's not possible to create flexbox of two items with total height equal the smaller one (with pure CSS at least). 

Was announcement message ever changed?